index.html

```diff
@@ -137,12 +136,39 @@
     </div>
   </script>
 
+  <!-- Project List template -->
+  <script type="text/template" id="project-list-template">
+    <%
+      var classNames = [
+        "project-list",
+        thumbnailView ? "thumbnail-view" : "",
+        starred ? "starred" : "",
+        featured ? "featured" : "",
+      ].join(" ");
+    %>
+    <div id="<%- id %>" 
+          class="<%- classNames %>" 
+          data-title="<%- title %>"
+          data-issue-online-privacy-and-security="<%- belongsToOnlinePrivacyAndSecurityIssue %>"
+          data-issue-open-innovation="<%- belongsToOpenInnovationIssue %>"
+          data-issue-decentralization="<%- belongsToDecentralizationIssue %>"
+          data-issue-web-literacy="<%- belongsToWebLiteracyIssue %>"
+          data-issue-digital-inclusion="<%- belongsToDigitalInclusionIssue %>" >
+      <div class="content">
+        <a href="<%- link %>" target="_blank"><h2><%- title %></h2></a>
```

Comment:
Title should be a link to the detail view of that project. It's like clicking on the "read more" button on a Project Card.